application/views/admin/survey/subview/tab_survey_view.php

```diff
@@ -25,7 +25,7 @@
 
     <!-- Create -->
     <li role="presentation" <?php if($active=='create'){echo 'class="active"';}?>>
-        <a data-toggle="tab" href='#general'>
+        <a role="tab" data-toggle="tab" href='#general'>
```

Comment:
Think we must add role="tabpanel" after http://accessibility.athena-ict.com/aria/examples/tabpanel2.shtml :)